src/Microsoft.Identity.Web.TokenCache/TokenCacheErrorMessage.cs

```diff
@@ -7,5 +7,6 @@ internal static class TokenCacheErrorMessage
     {
         public const string InitializeAsyncIsObsolete = "IDW10801: Use Initialize instead. See https://aka.ms/ms-id-web/1.9.0. ";
         public const string ExceptionDeserializingCache = "IDW10802: Exception occurred while deserializing token cache. See https://aka.ms/msal-net-token-cache-serialization general guidance and https://aka.ms/ms-id-web/token-cache-troubleshooting for token cache troubleshooting information.";
+        public const string CannotUseDistributedCache = "IDW10803: Cannot use distributed cache for the current configuration. Use an in memory cache instead.";
```

Comment:
You need to add it to the public API unshipped in all target frameworks (you can let the fixer do it, but you need to switch the target framework in VS)

Reply:
@jmprieur These changes are in the `Microsoft.Identity.Web.TokenCache` project, and there is only one set of shipped/unshipped files. There error isnt coming from this.

![image](https://github.com/user-attachments/assets/6f02abba-1bba-4068-a4d3-b0720405125d)

The error is from the fact that IdWeb is expecting a new API from MSAL which hasn't been released yet:
`TokenCacheNotificationArgs.NoDistributedCacheUseReason`

This is why the PR is still in draft mode. Once MSAL is released and IDWeb references the latest version, this PR will build successfully.